apps/measures/models.py

```diff
@@ -266,6 +266,18 @@ def get_foa_filter_info(self, reg, fut, inc, fac):
     def get_context(self, request):
         filtered_measures = self.measures_pages
 
+        # search
+        search = request.GET.get('search')
+        search_string = ''
+        if search:
+            searched_measures = filtered_measures.search(search)
+            search_string = _('Search for "{search_term}".').format(
+                search_term=search
+            )
+            filtered_measures = filtered_measures.filter(
+                id__in=[m.id for m in searched_measures]
+            )
+
```

Comment:
Is that sth else than just filtered_measures = searched_measures?

Reply:
searched_measures are of type ElasticSearchResults whereas filtered_measures is a query set. We can't `filter()` on the SearchResults